src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/RequesterMapper.java

```diff
@@ -1,14 +1,27 @@
 package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
 
+import java.util.Optional;
 import org.hl7.fhir.dstu3.model.Practitioner;
 import org.springframework.stereotype.Component;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
-
-import java.util.Optional;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.RequesterNameAndAddress;
 
 @Component
 public class RequesterMapper {
+    protected static final String SDS_USER_SYSTEM = "https://fhir.nhs.uk/Id/sds-user-id";
+
     public Optional<Practitioner> map(final Message message) {
-        return Optional.empty();
+        return message.getRequesterNameAndAddress()
+            .map(this::toPractitioner);
+    }
+
+    private Practitioner toPractitioner(final RequesterNameAndAddress requester) {
+        final var result = new Practitioner();
+        result.addIdentifier()
+            .setValue(requester.getIdentifier())
```

Comment:
I guess this is mandatory and will throw EdifactValidationException if missing?

Reply:
well according to the Pathology information model, the `identifier`, `role`, `speciality` and `name` are all mandatory